docker/Dockerfile

```diff
@@ -0,0 +1,47 @@
+FROM php:8.1-apache
+
+# Install system dependencies
+RUN apt-get update && apt-get install -y libonig-dev zip unzip
+
+# Install & Enable PHP extensions
+RUN docker-php-ext-install ctype iconv mbstring mysqli pdo_mysql
+RUN docker-php-ext-enable ctype iconv mbstring mysqli pdo_mysql
+RUN a2enmod headers rewrite
+
+# Install Composer
+RUN php -r "copy('https://getcomposer.org/installer', 'composer-setup.php');" \
+    && php composer-setup.php --install-dir=/usr/local/bin --filename=composer \
+    && php -r "unlink('composer-setup.php');"
+
+ENV COMPOSER_ALLOW_SUPERUSER=1
+
+# Clear cache
+RUN apt-get clean && rm -rf /var/lib/apt/lists/*
+
+WORKDIR /var/www/html
+
+# Copy existing application directory contents
+COPY ../. /var/www/html
+
```

Comment:
I *think* this should go last in the list of commands, so that this layer is added on after the other ones.  This way, if a dev makes a change to the application, eg, and wants to rebuild the container, the prior layers will be reused from cache.  I *think*.  It's been a while.